parquet/src/test/java/org/apache/iceberg/parquet/TestDictionaryRowGroupFilter.java

```diff
@@ -156,22 +156,22 @@ public class TestDictionaryRowGroupFilter {
   private MessageType parquetSchema = null;
   private BlockMetaData rowGroupMetadata = null;
   private DictionaryPageReadStore dictionaryStore = null;
-  private final WriterVersion writerVersion;
-
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
-
-  @Parameterized.Parameters
-  public static List<WriterVersion> writerVersions() {
-    return Arrays.asList(PARQUET_1_0, PARQUET_2_0);
+  @Parameter
+  private WriterVersion writerVersion;
+
+  @Parameters(name = "writerVersion={0}")
+  public static Collection<WriterVersion> parameters() {
+    return Arrays.asList(
+            WriterVersion.PARQUET_1_0,
+            WriterVersion.PARQUET_2_0);
   }
+  
+  @TempDir
+  public Path temp;
```

Comment:
can be private

Reply:
Solved in 6bfb542155b333a07341814e9ab65522e7596072